--- scribdfield.module	2010-01-10 15:56:32.000000000 -0600
+++ scribdfield.module.original	2010-01-10 12:20:24.000000000 -0600
@@ -46,7 +46,6 @@ function scribdfield_elements() {
   $filefield_elements = filefield_elements();
   $elements['scribdfield_widget'] = $filefield_elements['filefield_widget'];
   $elements['scribdfield_widget']['#process'][] = 'scribdfield_widget_process';
-//dpm($elements);
   return $elements;
 }
 
@@ -68,8 +67,6 @@ function scribdfield_widget_info() {
  * Implementation of CCK's hook_widget().
  */
 function scribdfield_widget(&$form, &$form_state, $field, $items, $delta = 0) {
-//dpm(array($form, $form_state, $field, $items, $delta));
-//dpm(filefield_widget($form, $form_state, $field, $items, $delta));
   return filefield_widget($form, $form_state, $field, $items, $delta);
 }
 
@@ -101,18 +98,7 @@ function scribdfield_widget_process($ele
       '#value'  => t('<div class="description">Files intended for Scribd.com are not sent until this entire form has been saved. Uploaded files are <strong>never</strong> automatically removed from Scribd.com: if you remove the file or delete this content, you\'ll also need to manually remove them from Scribd\'s "My Docs".</div>'),
     );
   }
-dpm(array('scribdfield_widget_process', $element));
-  $element['scribd_data'] = array(
-    'doc_id' => array(
-      '#type' => 'textfield',
-      '#title' => t('doc_id'),
-      '#default_value' => $node->doc_id,
-      '#description' => t('Scribd document ID. Required unless you are attaching a file.'),
-      '#required' => FALSE,
-      '#weight' => -5,
-    ), 
-  );
-dpm(array($element, $form, $form_state));
+
   return $element;
 }
 
@@ -126,7 +112,6 @@ function scribdfield_nodeapi(&$node, $op
         if (strpos($node_key, 'field_') !== FALSE) {
           $field = content_fields($node_key, $node->type);
           if ($field['widget']['module'] == 'scribdfield') {
-dpm($field); dpm($node);
             unset($_SESSION['scribfield_removed_files']);
             // see scribdfield_widget_process() for sets.
 
@@ -245,7 +230,6 @@ function scribdfield_form_content_field_
  * FormAPI theme function. Theme the output of an image field.
  */
 function theme_scribdfield_widget(&$element) {
-//dpm(array($element, theme('form_element', $element, $element['#children'])));
   return theme('form_element', $element, $element['#children']);
 }
 
